### DISCUSSION TITLE
Support for base URL / subfolder configuration

### DISCUSSION THREAD

**Discussion by goose-ws** [jonaswinkler/paperless-ng#679] [Feature Requests]
I'm hoping it wouldn't be too much trouble to add support for a base URL / sub-folder configuration.

What I mean by that is that instead of accessing my instance at `http://paperless.domain.com` I could access it at `http://domain.com/paperless`, and all the URL's would build off that `/paperless` base URL. For example, instead of `http://paperless.domain.com/dashboard` it could be `http://domain.com/paperless/dashboard`.

The reason being, it's much easier to reverse proxy an iframe, as I'm wanting to do with [Organizr](https://github.com/causefx/Organizr).

Thanks for any thought/consideration!

**Comment by jonaswinkler** [comment]
It actually is a lot of trouble, and I've tried that before. See #36.

**Reply by tandy-1000** [reply]
I just spent a while trying to figure this out, there's some mention of it being a possibility in the hosting section of the documentation, might be worth removing this line until that issue is closed:
`Override the STATIC_URL here. Unless you’re hosting Paperless off a subdomain like /paperless/, you probably don’t need to change this.`

Funnily enough I completely missed the point above it, `PAPERLESS_FORCE_SCRIPT_NAME=<path>` which mentions that changing the Base Path is untested.

**Reply by michaelkrieger** [reply]
> which mentions that changing the Base Path is untested.

Though it works wonderfully.
